pkg/kubectl/resource_printer.go

```diff
@@ -1749,6 +1749,9 @@ func printRoleBindingList(list *rbac.RoleBindingList, w io.Writer, options Print
 }
 
 func printClusterRole(clusterRole *rbac.ClusterRole, w io.Writer, options PrintOptions) error {
+	if options.WithNamespace {
+		return fmt.Errorf("clusterRole is not namespaced")
```

Comment:
It is a unified format of the message. 
e.g. 
`kubectl get ns --all-namespaces`
```
NAMESPACE   NAME      STATUS    AGE
namespace is not namespaced
namespace is not namespaced
```